scriptenginedebugger.h:34:10: fatal error: QScriptEngine: No such file or directory

I am getting the above error every time I compile without qtscript enabled 
(i.e. `-DCMAKE_DISABLE_FIND_PACKAGE_Qt5Script=ON` in the ebuild).
```
[ 58%] Building CXX object plugins/scriptenginedebugger/CMakeFiles/gammaray_scriptenginedebugger_plugin.dir/scriptenginedebugger.cpp.o
cd /tmp/portage/dev-util/gammaray-2.9.0/work/gammaray-2.9.0_build/plugins/scriptenginedebugger && /usr/bin/x86_64-pc-linux-gnu-g++ -DQT_CORE_LIB -DQT_DISABLE_DEPRECATED_BEFORE=0x040800 -DQT_NO_CAST_TO_ASCII -DQT_NO_DEBUG -DQT_NO_URL_CAST_FROM_STRING -DQT_SCRIPTTOOLS_LIB -DQT_STRICT_ITERATORS -DQT_USE_FAST_CONCATENATION -DQT_USE_FAST_OPERATOR_PLUS -Dgammaray_scriptenginedebugger_plugin_EXPORTS -I/tmp/portage/dev-util/gammaray-2.9.0/work/gammaray-2.9.0_build/plugins/scriptenginedebugger -I/tmp/portage/dev-util/gammaray-2.9.0/work/gammaray-2.9.0/plugins/scriptenginedebugger -I/tmp/portage/dev-util/gammaray-2.9.0/work/gammaray-2.9.0_build/plugins/scriptenginedebugger/gammaray_scriptenginedebugger_plugin_autogen/include -I/tmp/portage/dev-util/gammaray-2.9.0/work/gammaray-2.9.0 -I/tmp/portage/dev-util/gammaray-2.9.0/work/gammaray-2.9.0/3rdparty -I/tmp/portage/dev-util/gammaray-2.9.0/work/gammaray-2.9.0_build -isystem /usr/include/qt5 -isystem /usr/include/qt5/QtScriptTools -isystem /usr/include/qt5/QtCore -isystem /usr/lib64/qt5/mkspecs/linux-g++ -I/tmp/portage/dev-util/gammaray-2.9.0/work/gammaray-2.9.0_build/core -I/tmp/portage/dev-util/gammaray-2.9.0/work/gammaray-2.9.0_build/common   -DNDEBUG -march=native  -frecord-gcc-switches  -flto=5 -flto-report -fuse-linker-plugin  -pipe -O2 -Wunused-but-set-variable -Wlogical-op -Wsizeof-pointer-memaccess -Wreorder -Wformat-security -Wno-c++1z-extensions -Wall -Wextra -pedantic -Woverloaded-virtual -Winit-self -Wmissing-include-dirs -Wunused -Wundef -Wpointer-arith -Wmissing-noreturn -Werror=return-type -Wswitch -std=gnu++0x -fno-operator-names -Wzero-as-null-pointer-constant -fPIC -fvisibility=hidden -fvisibility-inlines-hidden   -fPIC -std=gnu++11 -o CMakeFiles/gammaray_scriptenginedebugger_plugin.dir/scriptenginedebugger.cpp.o -c /tmp/portage/dev-util/gammaray-2.9.0/work/gammaray-2.9.0/plugins/scriptenginedebugger/scriptenginedebugger.cpp
In file included from /tmp/portage/dev-util/gammaray-2.9.0/work/gammaray-2.9.0/plugins/scriptenginedebugger/scriptenginedebugger.cpp:29:0:
/tmp/portage/dev-util/gammaray-2.9.0/work/gammaray-2.9.0/plugins/scriptenginedebugger/scriptenginedebugger.h:34:10: fatal error: QScriptEngine: No such file or directory
 #include <QScriptEngine>
          ^~~~~~~~~~~~~~~
compilation terminated.
```

When I enable the script USE flag (`-DCMAKE_DISABLE_FIND_PACKAGE_Qt5Script=OFF`) no error occurs.

I found a workaround, if script is disabled a `sed` removes `add_subdirectory(scriptenginedebugger)` line from plugins/CMakeLists.txt in the ebuild.
```
if use !script ; then
	sed -i "/add_subdirectory(scriptenginedebugger)/d" plugins/CMakeLists.txt || die
fi
```
This is not the case for other config options, so am I missing another config option, or it's a bug in the CMakeLists?

--- a/plugins/CMakeLists.txt
+++ b/plugins/CMakeLists.txt
@@ -31,7 +31,9 @@ if(Qt5Widgets_FOUND OR QT_QTGUI_FOUND)
   add_subdirectory(styleinspector)
 endif()
 
-if(QT_QTSCRIPTTOOLS_FOUND OR Qt5ScriptTools_FOUND)
+# just checking whether Qt5::ScriptTools is found should be enough, but it isn't.
+# thus check both that Qt5::Script & Qt5::ScriptTools are found to fix https://github.com/KDAB/GammaRay/issues/492
+if((QT_QTSCRIPT_FOUND AND QT_QTSCRIPTTOOLS_FOUND) OR (Qt5Script_FOUND AND Qt5ScriptTools_FOUND))
   add_subdirectory(scriptenginedebugger)
 endif()
 

